test/run_pass/test_adc.js

```diff
@@ -16,9 +16,17 @@
 
 var adc = require('adc');
 var assert = require('assert');
-var stm32fPin = require('pin').STM32F4DIS;
+var testPin = null;
```

Comment:
The `testPin` variable is not used at all. Should this be used when creating an adc object?